docker-compose.yml

```diff
@@ -22,6 +22,7 @@ services:
   app:
     build: app
```

Comment:
it should be noted that 

```
build: app
```

will not work with sub-items. I reverted to my "old" few lines:

```
  app:
    build:
      context: app
      args:
        - edition=team
```